package/gluon-mesh-vpn-tunneldigger/files/usr/bin/tunneldigger-watchdog

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/lua
+
+function restart_tunneldigger()
+	io.popen('logger -t tunneldigger-watchdog "Restarting Tunneldigger."')
+	os.execute("/etc/init.d/tunneldigger restart")
+end
+
+function read_pid_file()
+	local pid_file = io.open("/var/run/tunneldigger.mesh-vpn.pid", "r")
+	if not pid_file then
+		return nil
+	end
+	pid = pid_file:read("*l") 
+	pid_file:close()
+	return pid
+end
+
+function check_for_mesh_vpn_neighbours()
```

Comment:
Oh, and renaming this function to something like `has_mesh_vpn_neighbours` would better reflect the return value IMHO.